code/datums/components/mob_chain.dm

```diff
@@ -42,11 +44,19 @@
 	RegisterSignal(parent, COMSIG_MOVABLE_MOVED, PROC_REF(on_moved))
 	RegisterSignal(parent, COMSIG_ATOM_CAN_BE_PULLED, PROC_REF(on_pulled))
 	RegisterSignals(parent, list(COMSIG_LIVING_UNARMED_ATTACK, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, COMSIG_MOB_ATTACK_RANGED), PROC_REF(on_attack))
+	RegisterSignal(parent, COMSIG_MOVABLE_UPDATE_GLIDE_SIZE, PROC_REF(on_glide_size_changed))
 	if (vary_icon_state)
 		RegisterSignal(parent, COMSIG_ATOM_UPDATE_ICON_STATE, PROC_REF(on_update_icon_state))
 		update_mob_appearance()
 	if (pass_damage_back)
-		RegisterSignal(parent, COMSIG_LIVING_ADJUST_DAMAGE, PROC_REF(on_adjust_damage))
+		RegisterSignals(parent, list(
+			COMSIG_LIVING_ADJUST_BRUTE_DAMAGE,
+			COMSIG_LIVING_ADJUST_BURN_DAMAGE,
+			COMSIG_LIVING_ADJUST_CLONE_DAMAGE,
+			COMSIG_LIVING_ADJUST_OXY_DAMAGE,
+			COMSIG_LIVING_ADJUST_TOX_DAMAGE,
```

Comment:
Hey why don't we just have a define-baked-in list of of all the types of damage

```dm
#define COMSIG_LIVING_ADJUST_ALL_STANDARD_DAMAGES list(whatever_shit)
```

that would be a nice substitute for `COMSIG_LIVING_ADJUST_DAMAGE`. We can leave stamina damage out of the define if you want